packages/ckeditor5-table/src/converters/tableproperties.js

```diff
@@ -17,9 +17,10 @@
  * @param {String} options.viewElement The view element name that should be converted.
  * @param {String} options.defaultValue The default value for the specified `modelAttribute`.
  * @param {Boolean} [options.reduceBoxSides=false]
+ * @param {Function} [options.isApllicable] The function which returns `true` if style should be checked for this element.
  */
 export function upcastStyleToAttribute( conversion, options ) {
-	const { viewElement, defaultValue, modelAttribute, styleName, reduceBoxSides = false } = options;
+	const { viewElement, defaultValue, modelAttribute, styleName, reduceBoxSides = false, isApplicable = () => true } = options;
```

Comment:
Maybe `checkElement` would be a better name?

Reply:
I think that `checkElement` tells less thank current name.